radixdlt/src/main/java/com/radixdlt/consensus/safety/WhitelistQuorum.java

```diff
@@ -0,0 +1,55 @@
+/*

```

Comment:
This seems to be a test class.  Should it move into tests?

Reply:
It's not a test class, it's a proper basic implementation of QuorumRequirements